autoload/suda.vim

```diff
@@ -89,19 +90,24 @@ function! suda#write(expr, ...) abort range
           \ options.cmdarg,
           \ tempfile,
           \))
-    let tee_cmd = 'tee'
     if has('win32')
       " In MS Windows, tee.exe has been placed at $VIMRUNTIME and $VIMRUNTIME
       " is added to $PATH in Vim process so `executable('tee')` returns 1.
       " However, sudo.exe executes a command in a new environment.  The
       " directory $VIMRUNTIME is not added here, so `tee` is not found.
       " Using a full path for tee command to avoid this problem.
-      let tee_cmd = exepath(tee_cmd)
+      let tee_cmd = exepath('tee')
+      let result = suda#system(
+            \ printf('%s %s', shellescape(tee_cmd), shellescape(path)),
+            \ join(readfile(tempfile, 'b'), "\n")
+            \)
+    else
+      let result = suda#system(printf(
+            \ 'dd if=%s of=%s bs=1048576 status=none',
```

Comment:
question

I'm sorry but I don't know what `status=none` is. Could you explain the behavior of that?

Reply:
`status=none` prevents dd from writing to stderr if no errors occur

Reply:
Oh, it seems that `status=none` is non-POSIX, so I will remove it.